TelePharo-Core.package/TlpRemoteUIManager.class/instance/registerRemoteTool.as..st

```diff
@@ -4,7 +4,13 @@ registerRemoteTool: aSeamlessProxy as: aNameSelector
 	| existingTool |
 	self shouldBeRegistered.
 	existingTool := Smalltalk tools remoteToolNamed: aNameSelector.
-	existingTool isSeamlessProxy ifFalse: [ "otherwise new remote tools are connected"
-		localTools at: aNameSelector ifAbsentPut: existingTool].
+	
+	existingTool isSeamlessProxy 
+		ifFalse: [ "otherwise new remote tools are connected" 
+			localTools at: aNameSelector ifAbsentPut: existingTool].
```

Comment:
I still do not get why following version does not work as you said. Seems I miss something obvious:
```Smalltalk
existingTool isSeamlessProxy 
		ifFalse: [ "otherwise new remote tools are connected" 
			localTools at: aNameSelector ifAbsentPut: existingTool]
                ifTrue: [self cleanupEnvironmentFromRemoteTool]
```